src/common/ConsoleMessage.ts

```diff
@@ -110,6 +110,13 @@ export class ConsoleMessage {
    * @returns The location of the console message.
    */
   location(): ConsoleMessageLocation {
-    return this._location;
+    return this._stackTraceLocations[0] ?? {};
```

Comment:
We [support](https://github.com/puppeteer/puppeteer#usage) the latest maintenance Node.js LTS, so we cannot use nullish coalescing syntax in the compiled JS yet. Let's double-check that the generated output works in Node.js v10. cc @jackfranklin

Reply:
If CI passes we should be good as that compiles and runs on Node 10. I think our ts-config is set up to compile this down correctly but we will see!

Reply:
the build artifacts in `lib/` still has `??`. Let me rephrase this in outdated JS.

Reply:
Looks like TypeScript is not compiling it away. I can look into fixing it or I'd rewrite to use a ternary (or similar) to avoid this for now, to be honest.

Reply:
Longer-term it might make sense to fix our tsconfig to align with the maintenance Node.js LTS version, so we don't have to worry about this in the future.